pkg/cli/admin/upgrade/status/status.go

```diff
@@ -261,7 +261,7 @@ func (o *options) Run(ctx context.Context) error {
 		if message = progressing.Message; message == "" {
 			message = "<none>"
 		}
-		fmt.Fprintf(o.Out, "The cluster version is not updating (%s=%s).\n\n  Reason: %s\n  Message: %s\n", progressing.Type, progressing.Status, reason, strings.ReplaceAll(message, "\n", "\n  "))
+		fmt.Fprintf(o.Out, "The cluster is not updating.")
```

Comment:
We can remove the logic regarding the `reason` and `message` variables in this condition as they are not used.

```go
	if progressing.Status != configv1.ConditionTrue && !isWorkerPoolOutdated {
		fmt.Fprintf(o.Out, "The cluster is not updating.\n")
		return nil
	}
```